pkg/secrethub/client.go

```diff
@@ -19,15 +20,29 @@ type Client interface {
 	Users() UserService
 }
 
+// Decryptor decrypts data, typically an account key.
+type Decryptor interface {
+	// Unwrap decrypts data, typically an account key.
+	Unwrap(ciphertext crypto.CiphertextRSAAES) ([]byte, error)
```

Comment:
How does this work when the `Decryptor` does not decrypt `RSAAES` ciphertexts?

Reply:
For now, decrypters only decrypt account keys, which are `RSAAES` ciphertexts.

We can change the interface to accept bytes when we return the account key as bytes.